tests/virt_autotest/login_console.pm

```diff
@@ -110,7 +121,7 @@ sub login_to_console {
         }
     }
 
-    if (!check_screen([qw(grub2 grub1 prague-pxe-menu)], 210)) {
+    unless (!check_screen([qw(grub2 grub1 prague-pxe-menu)], 210) or is_tumbleweed) {
```

Comment:
The logic here changes ? If it is SLES and check_screen fails, ipmitool power reset will be triggered previously, but now if check_screen succeeds, system will reboot ?

Reply:
good catch. forgot to remove "!" when 'if' was changed to 'unless'.